src/js/components/Breadcrumb.js

```diff
@@ -124,43 +120,37 @@ class Breadcrumb extends React.Component {
     return lastItemLink.scrollWidth + LAST_ITEM_OFFSET;
   }
 
-  getCurrentNamedRoutes() {
-    let {router} = this.context;
-    let namedRoutes = {};
-    let routes = router.getCurrentRoutes();
-
-    routes.forEach(function (route) {
-      // Strip regex characters from route.path
-      let path = route.path.slice().replace(/\?/g, '').replace(/\/\//g, '/');
-      if (path.charAt(0) === '/') {
-        path = path.slice(1);
-      }
-      if (path.charAt(path.length - 1) === '/') {
-        path = path.slice(0, -1);
-      }
-      // If named path already exists, don't overwrite. Sometimes root path
-      // returns a named route match and an unnamed match.
-      if (!(path in namedRoutes) || (namedRoutes[path] == null)) {
-        namedRoutes[path] = route.name;
-      }
-    });
-
-    return namedRoutes;
-  }
-
   getCurrentRouteParams() {
     // Isolated for easier testing
     return this.context.router.getCurrentParams();
   }
 
-  getCrumb(label, route, params, key) {
-    return (
-      <li key={key}>
-        <Link to={route}
-            params={params}
+  getCrumb(crumb, key) {
+    let content = crumb;
```

Comment:
This assignment looks a little bit weird because, you still using `crumb` directly below.
I think you can get rid of `content`.